frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import pytest
+import shakedown
+
+
+@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = url.split('/')[4] # TODO: does this need to be a random string?
```

Comment:
My suspicion is that the `package_name` used in the `dcos repo add` command is more correctly called the _repo_name_, and the installed repo provides a well-known package name.